frontend/src/metabase-types/analytics/simple-event.ts

```diff
@@ -0,0 +1,13 @@
+type SimpleEventSchema = {
+  event: string;
+  target_id?: number | null;
+  triggered_from?: string | null;
+  duration_ms?: number | null;
+  result?: string | null;
+  event_detail?: string | null;
+};
+
+export type ValidateSimpleEvent<
+  T extends SimpleEventSchema &
+    Record<Exclude<keyof T, keyof SimpleEventSchema>, never>,
+> = T;
```

Comment:
I moved `ValidateSimpleEvent` out so we can use it from other files without circular imports.